addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4682,19 +4678,11 @@ registry.sizing = SnippetOptionWidget.extend({
             $handle.toggleClass('o_handle_end', current === classes.length - 1);
         }
 
-        // Adapt the handles to fit the left, top and bottom sizes
-        var ml = this.$target.css('margin-left');
-        this.$overlay.find('.o_handle.w').css({
-            width: ml,
-            left: '-' + ml,
-        });
-        this.$overlay.find('.o_handle.e').css({
-            width: 0,
-        });
-        this.$overlay.find(".o_handle.n, .o_handle.s").toArray().forEach((handle) => {
+        // Adapt the handles to fit top and bottom sizes
+        this.$overlay.find(".o_handle.n:not(.o_grid_handle), .o_handle.s:not(.o_grid_handle)").toArray().forEach((handle) => {
             var $handle = $(handle);
             var direction = $handle.hasClass('n') ? 'top' : 'bottom';
-            $handle.height(self.$target.css('padding-' + direction));
+            $handle.outerHeight(self.$target.css('padding-' + direction));
```

Comment:
When `.height` is used, the handle area overlaps the content according to the size of its border (here, 2px).
Using `.outerHeight` solves this issue.
| `.height`  | `.outerHeight` |
| ------------- | ------------- |
| ![Capture d’écran 2023-09-21 à 15 01 39](https://github.com/odoo/odoo/assets/80679690/20fa1759-ce6a-4f01-8ab9-d0d20df1dd2c)  | ![Capture d’écran 2023-09-21 à 15 07 06](https://github.com/odoo/odoo/assets/80679690/4c3e69e2-0a32-4595-99b1-8b8034234822)  |

Reply:
Well spotted :+1: But then I think I am seeing the same thing in stable version :fearful: For a one line change, I would have considered merging that in stable.

But let's say we should not because it is not a critical problem. However, what you should always try to do is making that an independent commit. So, here I'd say: make this a dedicated commit in this same PR (that you can label with [FIX]). This is important because I am asking this question now but if I had not seen that you made this change and were to wonder "why `outerHeight`?" in two years, I would blame that line and found out your [IMP] commit which does not explain why we had to use `outerHeight` instead of height :wink:

Reply:
Done 👍 f9a603399349a36f7ab701ced19a341eada0850e